src/pyodide/internal/patches/random.py

```diff
@@ -0,0 +1,27 @@
+import random as _orig_random
```

Comment:
```suggestion
# Wraps the `random` module so that calls to any methods within it raise
# unless they are called as part of a request.
import random as _orig_random
```